src/main/features/response/views/confirmation/part-admission.njk

```diff
@@ -0,0 +1,25 @@
+{% set claimantName = claim.claimData.claimant.name %}
+{% set claimantDetailsPageURI = ClaimPaths.claimantDetailsPage.evaluateUri({ externalId: claim.externalId }) %}
+{% set amount = claim.response.paymentDetails.amount| numeral %}
+
+{% if claim.response['isAlreadyPaid'] === 'yes' %}
+  <p> {{ t('You told us you’ve paid the {{ amount }} you believe you owe. We’ve sent {{ claimantName }} this response.', { claimantName: claimantName, amount: amount }) }}</p>
+{% endif %}
+
+{% if claim.response.paymentOption %}
+  {% if claim.response.paymentDate %}
+    {% set paymentDate = claim.response.paymentDate | date %}
+  {% endif %}
+  {% if claim.response.paymentOption === domain.PaymentOption.IMMEDIATELY %}
+    {% set paymentMethod = 'immediately' %}
```

Comment:
Isn't it `domain.PaymentOption.IMMEDIATELY.displayValue`?

Reply:
We don't have display value in this object (`src/main/app/claims/models/response/core/paymentOption.ts`), so can't do that